### ISSUE TITLE
loss 下降到2.7就不下降了

### ISSUE THREAD

**Issue by Alexender-Ye** [FlagOpen/FlagEmbedding#744]
你好，我在做一个QA的数据集的任务，其中query是question，pos是answer，neg是其他的选项，但是在训练的时候，loss下降到2.7就不再下降了，并且最后accuracy也大概只有20-30%，请问大佬知道这种事什么问题么？

**Comment by staoxiao** [comment]
The loss seems large. I guess there are pseudo-negative examples in the training data (some samples in the negative sample list `neg: List[str]` are actually positive samples).
